examples/04_scalability.py

```diff
@@ -197,3 +201,49 @@ def wrapped_func(*args, **kwargs):
 [t.set(size=16) for t in ax2.get_yticklabels()]
 plt.tight_layout()
 
+################################################################################
+# Similarity encoder with most frequent strategy and float32 dtype
+# ----------------------------------------------------------------
```

Comment:
I would use as a title "Reduce memory usage during encoding using float32".

I would add a sentence stressing that the subsequent scikit-learn model may upcast to float64, but that it leads to memory saving during the encoding, as can be seen below.